fluent-plugin-enhance-k8s-metadata/lib/sumologic/kubernetes/connector.rb

```diff
@@ -18,17 +18,24 @@ def core_clients
       end
 
       def group_clients
-        @api_groups.map do |ver|
-          [ver, create_client('apis', ver)]
+        @api_groups.map do |elem|
+          # elem should have base as key e.g. 'extensions' and version as values e.g. 'v1beta1'
+          if elem.length < 2
+            continue
+          end
+
+          base = sprintf "%s", elem[0]
```

Comment:
why not `elem[0].to_s`?

Reply:
🤷🏻 

Is it better? (no idea) I can change it